dsb-gui/src/test/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/FileTreeTableItemTest.java

```diff
@@ -83,23 +68,19 @@ public void refresh() throws Exception {
         new JFXPanel();
         Platform.runLater(() -> {
             try {
-                final ClassLoader classLoader = Ds3GetJob_Test.class.getClassLoader();
-                final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER);
-                Path path = null;
-                if (url != null) {
-                    path = new File(url.getFile()).toPath();
-                }
+                final Path path = ResourceUtils.loadFileResource(SessionConstants.LOCAL_FOLDER);
+                final File testFolder = path.toFile();
+                final int numFiles = testFolder.list().length;
                 final FileTreeTableProvider fileTreeTableProvider = new FileTreeTableProvider();
                 final FileTreeModel fileTreeModel = new FileTreeModel(path, FileTreeModel.Type.Directory, 0, 0, "");
                 final FileTreeTableItem fileTreeTableItem = new FileTreeTableItem(fileTreeTableProvider, fileTreeModel, new Workers());
                 fileTreeTableItem.refresh();
                 final ObservableList<TreeItem<FileTreeModel>> children = fileTreeTableItem.getChildren();
-                if (children.size() == path.toFile().list().length) {
+                if (children.size() == numFiles) {
                     successFlag = true;
                 }
                 latch.countDown();
-            }
-            catch (final Exception e) {
+            } catch (final Exception e) {
                 e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then re-throw it and let junit deal with it.  Leave the latch.countDown() though.